src/comic-page/entities/comic-page.dto.ts

```diff
@@ -30,6 +31,13 @@ export async function toComicPageDto(page: ComicPage) {
   return pageDto;
 }
 
+export async function toSortedComicPageDto(pages: ComicPage[]): Promise<ComicPageDto[]> {
+  return sortBy(
+    await Promise.all(pages.map(async (page) => toComicPageDto(page))),
+    'pageNumber',
+  );
+}
+
```

Comment:
Move this sort logic to the `toComicPageDtoArray` as it's a default strategy whenever we're fetching an array of pages